kinds/dashboard/dashboard_kind.cue

```diff
@@ -200,6 +200,8 @@ lineage: schemas: [{
 			current?: #VariableOption
 			// Whether multiple values can be selected or not from variable value list
 			multi?: bool | *false
+			// Allow custom values to be entered in the variable
+			allowCustomValue?: bool | *true
```

Comment:
we want to have this default to true to maintain how variables worked before this PR. When creating a new var, we implicitly allowed adding custom vars